packages/@coorpacademy-app-player/src/middlewares/test/error-logger.js

```diff
@@ -0,0 +1,54 @@
+import test from 'ava';
+import {createStore} from 'redux';
+import createMiddleware from '../index';
+import macro from '../../actions/test/helpers/macro';
+
+test('should call ErrorLogger', async t => {
```

Comment:
`should call Logger.error when action.error is true`.

Rajoute un test sur le fait que Logger.error n'est pas appelé lorsque `action.error` st false, et idem undefined.

Reply:
action.error false et undefined c'est pas la même chose ? ça peut faire le même scenario, non ?